go-controller/pkg/cni/types/types.go

```diff
@@ -45,6 +45,15 @@ type NetConf struct {
 	LogFileMaxAge int `json:"logfile-maxage"`
 }
 
+// NetConfList describes an ordered list of NetConf
+type NetConfList struct {
+	CNIVersion string `json:"cniVersion,omitempty"`
+
+	Name         string     `json:"name,omitempty"`
+	DisableCheck bool       `json:"disableCheck,omitempty"`
+	Plugins      []*NetConf `json:"plugins,omitempty"`
+}
+
```

Comment:
hm, can't we just import the libcni `NetConfList` type ? https://github.com/containernetworking/cni/blob/f6506e215fb27ffbd9b46d9d18aae29dba697a52/pkg/types/types.go#L78

Reply:
we can't because it is our `NetConf` not their
`	Plugins      []*NetConf `json:"plugins,omitempty"``

Reply:
right. 

so I would recommend:
- remove disable check. We don't even implement command check iiuc.
- have name & cni version required. According to our implementation, it seems they are. 

Maybe it is better to wait for an OVN-K CNI maintainer ?....

@dcbw could you chime in this one ?

Reply:
if we make name and cni version required, we left with just plugins, maybe make it required as well ?
this will make the parsing more deterministic, if plugins is there , all 3 fields must exists, otherwise it is a "single"
wdyt?

Reply:
sounds good IMO. 

Never the less, let's wait for a maintainer's opinion.

Reply:
not relevant anymore due to 
https://github.com/ovn-org/ovn-kubernetes/pull/3844/files#r1371651947
(removed the struct)